.github/workflows/phpunit.yml

```diff
@@ -8,5 +8,5 @@ on:
 
 jobs:
   PHPUnit:
-    uses: discoverygarden/phpunit-action/.github/workflows/phpunit.yml@v1
+    uses: discoverygarden/phpunit-action/.github/workflows/phpunit.yml@fix/auth
```

Comment:
Should be pointed at whatever release after https://github.com/discoverygarden/phpunit-action/pull/11 gets merged.